foundations/javascript_basics/problem_solving.md

```diff
@@ -1,61 +1,68 @@
 
 In this lesson we will walk through a few techniques that can be used to help with the problem solving process.
 
 ### Learning Outcomes
+
 By the end of this lesson, you should be able to do the following:
 
-* Explain the three steps in the problem solving process.
-* Explain what pseudo code is and be able to use it to solve problems.
-* Be able to break a problem down into subproblems.
+- Explain the three steps in the problem solving process.
+- Explain what pseudo code is and be able to use it to solve problems.
+- Be able to break a problem down into subproblems.
 
 ### Understand the Problem
+
 <span id="problem-solving-stages"></span>
 The first step to solving a problem is understanding exactly what the problem is.<span id="important-understand-problem"> If you don't understand the problem you won't know when you've successfully solved it and may waste a lot of time on a wrong solution</span>.
 
 <span id="help-understand-problem">To gain clarity and understanding of the problem, write it down on paper, reword it in plain English until it makes sense to you, and draw diagrams if that helps. When you can explain the problem to someone else in plain English, you understand it.</span>
 
 ### Plan
+
 Now that you know what you're aiming to solve, don't jump into coding just yet. It's time to plan out how you're going to solve it first.
-<span id="planning-stage">
+<span id="planning-stage"></span>
 Some of the questions you should answer at this stage of the process:
 
-* Does your program have a user interface? What will it look like? What functionality will the interface have? Sketch this out on paper.
-* What inputs will your program have? Will the user enter data or will you get input from somewhere else?
-* What's the desired output?
-* Given your inputs, what are the steps necessary to return the desired output?
+- Does your program have a user interface? What will it look like? What functionality will the interface have? Sketch this out on paper.
+- What inputs will your program have? Will the user enter data or will you get input from somewhere else?
+- What's the desired output?
+- Given your inputs, what are the steps necessary to return the desired output?
 
 The last question is where you will write out an algorithm to solve the problem. <span id="algorithm">You can think of an algorithm as a recipe for solving a particular problem. It defines the steps that need to be taken by the computer to solve a problem in pseudo code.</span>
-</span>
-#### Pseudo Code
+
+### Pseudo Code
+
 <span id="pseudo">Pseudo code is writing out the logic for your program in natural language instead of code. It helps you slow down and think through the steps your program will have to go through to solve the problem.</span>
 
 Here's an example of what the pseudo code for a simple program that prints all numbers up to an inputted number might look like:
 
-~~~
+```
```

Comment:
We use tildes for our markdown.

Reply:
I had prettier activated. I reverted the changes made by prettier and made a new commit.